GUI/SearchTaskWindow.xaml

```diff
@@ -240,7 +464,15 @@
                                     </Grid.ColumnDefinitions>
                                     <StackPanel Grid.Column="0">
                                         <StackPanel Orientation="Horizontal" Margin="1">
-                                            <CheckBox x:Name="allAmbiguity" Content="Report PSM ambiguity" Margin="5" />
+                                            <CheckBox x:Name="allAmbiguity" Content="Report PSM ambiguity" Margin="5" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                                <CheckBox.ToolTip>
+                                                    <TextBlock>
+                                                        Database searching occasionally cannot discriminate between multiple sequences (such as XXIXX and XXLXX).
```

Comment:
Multiple peptides are reported when the peptide identification in a PSM is ambiguous (e.g., the sequences PEPTIDE and PEPTLDE). If this is left unchecked, a peptide is randomly chosen among a PSM's options in case of a tie.